ServerCore/Areas/Identity/UserAuthorizationPolicy/IsEventAuthor.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc.Filters;
+using ServerCore.DataModel;
+
+namespace ServerCore.Areas.Identity.UserAuthorizationPolicy
+{
+    /// <summary>
+    /// Require that the current user is an author for the event in the route.
+    /// </summary>
+    public class IsAuthorInEventRequirement : IAuthorizationRequirement
```

Comment:
What's an author-only page?

Reply:
The general edit page for the module? Not sure but it seems weird to not include it.